mmtrack/apis/train.py

```diff
@@ -32,6 +31,11 @@ def train_model(model,
     """
     logger = get_root_logger(cfg.log_level)
 
+    if cfg.get('USE_MMDET', False):
```

Comment:
We can directly use the "build_dataset" in mmdet, do not need to turn to mmcls. 
You can try to register the dataset into the "DATASET" in mmdet. 
The same to your modifications below.